bootstrap/Dockerfile

```diff
@@ -86,20 +86,22 @@ ARG registries
 
 COPY $registries /opt/registries
 
-COPY --from=bootstrap_base ${GOPATH}/src/github.com/kubeflow/kubeflow/bootstrap/bin/bootstrapper /opt/kubeflow/
+COPY --from=bootstrap_base /go/src/github.com/kubeflow/kubeflow/bootstrap/bin/bootstrapper /opt/kubeflow/
```

Comment:
${GOPATH} should be safer?

Reply:
somehow it could not locate the file:
```
Step 33/44 : COPY --from=bootstrap_base ${GOPATH}/src/github.com/kubeflow/kubeflow/bootstrap/bin/bootstrapper /opt/kubeflow/
COPY failed: stat /usr/local/google/docker/overlay2/2ed96fd138f735c3c95d4a79a3cba2ae81cc5cbd47ae88772743473bf1057b3f/merged/src/github.com/kubeflow/kubeflow/bootstrap/bin/bootstrapper: no such file or directory
```